portfolio/src/main/webapp/script.js

```diff
@@ -98,28 +104,53 @@ function showBlogPost(postNum) {
 function hideBlogPost(postNum) {
   // Find the correct triangle to hide and switch its image.
   // Also change the onclick function for this button to show the post.
-  const buttonId = "expandTriangle" + postNum;
-  let button = document.getElementById(buttonId);
-  console.log(button);
+  const buttonId = 'expandTriangle' + postNum;
+  const button = document.getElementById(buttonId);
   if (button !== null) {
-    button.src = "/images/righttriangle.jpeg";
+    button.src = '/images/righttriangle.jpeg';
     button.onclick = function() {
       showBlogPost(postNum);
     };
-  }  
+  }
 
   // Remove blog post content from its area.
-  const blogPostId = "blogPostArea" + postNum;
-  let postArea = document.getElementById(blogPostId);
+  const blogPostId = 'blogPostArea' + postNum;
+  const postArea = document.getElementById(blogPostId);
   if (postArea !== null) {
     if (postNum > 0 && postNum <= posts.length) {
-      const elements = posts[postNum-1]
-      for (i = 0; i < elements.length; i++) {
-        const element = elements[i]
+      const elements = posts[postNum - 1];
+      let i = 0;
+      for (i; i < elements.length; i++) {
+        const element = elements[i];
         if (element !== null) {
           postArea.removeChild(element);
```

Comment:
So, it looks like you're creating a toggle to allow showing and hiding posts.

Rather than creating and destroying the HTML elements (expensive), why not just make them visible/invisible, using the right css attribute? (possibly visibility, but you'll need to google for the right name).

As I mentioned above, you should have your posts sitting in a container (e.g. `<article id="blog-post-1">...</article>`), and then you should just have a two-liner that might look like:

```
blogPost = document.getElementById(<your post id>);
blogPost.setAttribute('visibility', 'no');
```

(note: I am totally making up the methods above, just typing that out to show you what I mean).